### DISCUSSION TITLE
Import Marley Spoon PDF recipes

### DISCUSSION THREAD

**Discussion by Tscherno** [mealie-recipes/mealie#4492] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I'm using Marley Spoon a lot to order ready-to-cook meals over the last years. They offer the recipis as downloadable PDF. To transfer my favorite recipes to mealie i have to do a lot of manual copy-pasting and screenshotting.

### Please provide a concise description of the feature that would resolve your issue.

It would be cool if the PDFs could be directly imported into Mealie!

Sample PDFs:
[Brazilian-Style Chicken with Shrimp and Coconut.pdf](https://github.com/user-attachments/files/17604033/Brazilian-Style.Chicken.with.Shrimp.and.Coconut.pdf)

### Please consider and list out some caveats or tradeoffs made in your design decision

Language dependend. The recipe itself is translated, but the headers are in my language (german in that case).

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Mealie _almost_ supports this with the OpenAI integration. You can import a recipe via image, [more on that in the docs](https://docs.mealie.io/documentation/getting-started/installation/open-ai/).

However this obviously still requires some manual effort since we don't directly support PDFs. I agree this makes sense to build into the OpenAI integration.

**Comment by Tscherno** [comment]
A few more examples:
[Broccoli & feta lasagne with mushroom & grated courgette.pdf](https://github.com/user-attachments/files/17604063/Broccoli.feta.lasagne.with.mushroom.grated.courgette.pdf)
[Quinoa-Bowl mit Räucherlachs und Sriracha-Mayonnaise.pdf](https://github.com/user-attachments/files/17604069/Quinoa-Bowl.mit.Raucherlachs.und.Sriracha-Mayonnaise.pdf)

**Comment by ryanwinter** [comment]
What about just importing from their website? I tried to import this and it was a total fail :(

https://marleyspoon.com/menu/322123-baked-sweet-potatoes-with-spiced-chickpeas-tahini-dressing-cranberries
